scripts/run_lifecycle_soak.py

```
@@ -0,0 +1,410 @@
+            ) as key:
+                snapshots = {value: snapshot_value(key, value) for value in VALUE_NAMES}
+                key_states.append({"path": path, "existed": existed, "snapshots": snapshots})
+
+                winreg.SetValueEx(key, "DumpFolder", 0, winreg.REG_EXPAND_SZ, str(dump_dir))
+                winreg.SetValueEx(key, "DumpType", 0, winreg.REG_DWORD, 1)
+                winreg.SetValueEx(key, "DumpCount", 0, winreg.REG_DWORD, 10)
+
+        # The control's promise is a fail-fast termination, not merely a nonzero status: a control that exited any
+        # other way did not exercise the path WER is being asked to capture. Python reports the raw process exit
+        # status, so the exact NTSTATUS is comparable here.
+        control = subprocess.Popen([str(control_probe), "wer-crash"])
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)